src/easylink/pipeline_schema_constants/development.py

```diff
@@ -139,7 +139,21 @@
                     parent_slot="step_4_main_input",
                     child_node="step_4a",
                     child_slot="step_4a_main_input",
-                )
+                ),
+                SlotMapping(
+                    slot_type="input",
+                    parent_node="step_4",
+                    parent_slot="step_4_secondary_input",
+                    child_node="step_4a",
+                    child_slot="step_4a_secondary_input",
+                ),
+                SlotMapping(
```

Comment:
Shy is this repeated?

Reply:
If i understand you correctly, I think the answer is  that one is "4a" and one is "4b"?